package.json

```diff
@@ -135,6 +134,7 @@
 		"@babel/preset-env": "^7.10.1",
 		"@commitlint/cli": "^17.0.3",
 		"@commitlint/config-conventional": "^14.1.0",
+		"@kiva/kv-tokens": "^2.6.0",
```

Comment:
The update is good here, please add it back to the standard `dependencies` block instead of `depDeps`

Reply:
Thanks for consulting!

Reply:
Looks like this dependency still needs to return to non-dev-dependencies.